pkg/clients/clients.go

```diff
@@ -167,6 +173,8 @@ func New(kubeconfig string) *Settings {
 	clientSet.K8sClient = kubernetes.NewForConfigOrDie(config)
 	clientSet.VeleroClient = veleroClient.NewForConfigOrDie(config)
 	clientSet.VeleroV1Interface = veleroV1Client.NewForConfigOrDie(config)
+	clientSet.ClientCgu = clientCgu.NewForConfigOrDie(config)
+	clientSet.ClusterGroupUpgradeInterface = clientCguV1.NewForConfigOrDie(config)
```

Comment:
Please replace it by: 

`clientSet.RanV1alpha1Interface = clientCguV1.NewForConfigOrDie(config)`

Reply:
Updated